INSTALL.md

```diff
@@ -175,53 +183,17 @@ NB: to access the admin panel visit [localhost:8000/admin](http://127.0.0.1:8000
 
 ## Loading initial data (fixtures)
 
-[Providing initial data for models](https://docs.djangoproject.com/en/dev/howto/initial-data/)
-
-
-To initialize the database with the `saskatoon.json` dump file:
-```
-(venv)$ python3 saskatoon/manage.py migrate --skip-check
-(venv)$ python3 saskatoon/manage.py loaddata saskatoon/fixtures/saskatoon.json
-```
-
-Alternatively you could audit/modify the individual .json files located in `saskatoon/fixtures` and run:
+To initialise the database:
+1. Audit/modify the individual `.json` files located in `saskatoon/fixtures`
```

Comment:
Since we point to another README for more information on fixtures I would remove step 1. and simply show the init command.

Reply:
good call. fixed!